### PR TITLE
feat: support ts and update deps

### LINKED ISSUE #81: Typescript support
Why did you remove support for `.ts` files?

### PR DESCRIPTION
**Pull request by iChenLei**
try to fix #81 , and update ci config

### CODE REVIEW

**Comment by clinyong** [comment]
Why remove the `yarn.lock` file?

**Comment by iChenLei** [comment]
> Why remove the `yarn.lock` file?

fixed now.

File: tsconfig.json [hunk 1]
```diff
@@ -17,6 +17,7 @@
 	"exclude": [
 		"node_modules",
 		".vscode-test",
-        "out"
+        "out",
+		"src/test/fixtures" // #85
```
Comment (iChenLei) [inline tsconfig.json:21]: https://github.com/clinyong/vscode-css-modules/actions/runs/4762097006/jobs/8463968926#step:5:12
```bash
Error: src/test/fixtures/ts.ts(1,20): error TS2307: Cannot find module './sample.css' or its corresponding type declarations.
Error: src/test/fixtures/tsx.tsx(1,20): error TS2307: Cannot find module './sample.css' or its corresponding type declarations.
```

File: package.json [hunk 2]
```diff
@@ -52,13 +53,14 @@
         "prepare": "husky install"
     },
     "devDependencies": {
+        "@types/fs-extra": "^9.0.11",
```
Comment (iChenLei) [inline package.json:56]: I think `@types/fs-extra` is part of devDependencies?

**Review by clinyong — APPROVED** [review]

**Pull request merged**